libr/socket/run.c

```diff
@@ -286,6 +286,8 @@ static void setASLR(RRunProfile *r, int enabled) {
 #endif
 }
 
+#if __APPLE__ && !__POWERPC__
```

Comment:
are we sure that this does not break anything?

Reply:
yes, there were warnings in macos about not used `handle_redirection_proc` and `restore_saved_fd`. Those functions are used only in `handle_redirection` and `handle_redirection` is just a `return 0;` in this case, see https://github.com/radareorg/radare2/pull/17781/files#diff-91ebf3f9d4c19931970b6445d178761e435da9628a060c655969519191eee820R390

Reply:
okok. good.